admin/admin.go

```diff
@@ -0,0 +1,179 @@
+package admin
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/vocdoni/blind-csp/model"
+	"go.vocdoni.io/dvote/httprouter"
+	"go.vocdoni.io/dvote/httprouter/bearerstdapi"
+	"go.vocdoni.io/dvote/log"
+)
+
+const bearerTokenReqAmount = 100000000
+
+type Admin struct {
```

Comment:
All Exported methods, constants, types or vars should have a documentation comment `// Admin is the type....`

Now you have Github copilot that helps you write all these comments on a fly.